### ISSUE TITLE
DFC Icon in json data?

### ISSUE THREAD

**Issue by taw** [mtgjson/mtgjson#228]
Vaguely related to https://github.com/mtgjson/mtgjson/issues/227

As only vaguely related issue, DFC icon (day/night, day/eldlazi, whatever-that-is/planeswalker) could also be indicated in json somehow.

[711.1a / 711.1b](http://mtg.wtf/help/rules#section-711) have their official names.

It used to be simply sun/moon, but now it got a bit more complicated.

**Comment by csuhta** [comment]
Right now, you can determine this programmatically (until Wizards prints a new set and ruins it, of course)

If the layout is `transform`: 
- and the card has a `planeswalker` type on one side:
    - the front is a hero symbol, and the back is a planeswalker symbol
- and the card has an `eldrazi` type on one side:
    - the front is a eldritch moon, and the back is an eldrazi symbol
- otherwise the front is a day symbol and the back is a night symbol

**Comment by taw** [comment]
Yeah, either that, or just one rule per set, with exception of one EMN werewolf.

**Comment by randymorris** [comment]
@csuhta Sorry for digging up this thread but I just wanted to note that this algorithm won't work as is because of [Arlinn Kord](http://gatherer.wizards.com/Pages/Card/Details.aspx?multiverseid=410007).
